config.dev.toml

```diff
@@ -122,12 +122,12 @@ HGRCPath = ""
 # TraceExporterURL is the URL to which Athens populates distributed tracing
 # information such as Jaeger. In Stackdriver, use this as the GCP ProjectID.
 # Env override: ATHENS_TRACE_EXPORTER_URL
-TraceExporterURL = ""
+TraceExporterURL = "http://localhost:14268"
```

Comment:
Does this make Athens error out and exit if I don't have the Jaeger server running?

Reply:
No, it does not exit but we already have the jaeger in helm charts, so, I thought it might be nice to add it anyway.

Reply:
I read it the other way :(. Sorry about it!